src/components/Game.js

```diff
@@ -0,0 +1,251 @@
+import { h } from 'preact'
+import { Entity } from 'aframe-react'
+import { Presenter } from 'microcosm-preact'
+import {
+  flick,
+  ready,
+  reset,
+  startTimer,
+  stopTimer,
+  toggleSettings
+} from '../actions/game'
+import config from '../config.js'
+
+import Assets from './Assets'
+import MainScene from './MainScene'
+import Moves from './Moves'
+import Particles from './Particles'
+import Sky from './Sky'
+import Timer from './Timer'
+
+export default class Game extends Presenter {
+  getModel() {
+    return {
+      games: state => state.game.games,
+      hasWon: state => state.game.hasWon,
+      lights: state => state.game.lights,
+      ready: state => state.game.ready,
+      moves: state => state.game.moves,
+      textures: state => state.game.textures,
+      timeElapsed: state => state.game.timeElapsed
+    }
+  }
+
+  handleClick = e => {
+    // We wanna trigger a rotation on the target object here.
+    // Seems to be disconnected from the AFRAME instance.
+    // e.target.sceneEl.object3D.rotation.set(50, 90, 180)
+
+    if (this.model.moves === 0) {
+      this.repo.push(startTimer)
+    }
+
+    const payload = {
+      x: parseInt(e.target.attributes[0].value),
+      y: parseInt(e.target.attributes[1].value)
```

Comment:
Due to a-frame's declarative nature using HTML attributes, I have to use attributes to bind data. So, I declare each light as `<Entity x={x} y={y} {...props}`. Then, when the click event fires on any given light, I'm able to read those attributes back again when dispatching the action. I'm sure there's a better way but this seems to work fine for now!